zk/src/archive/web/js/zk/mount.js

```diff
@@ -97,6 +97,9 @@ function zkamn(pkg, fn) {
 					_paci = null;
 					for (var f; f = fs.shift();)
 						f();
+					if (zk._fireOnClientInfo) {
```

Comment:
Once `zk._fireOnClientInfo` is set, it always exists. So reaching this always triggers `onClientInfo` .
Set `zk._fireOnClientInfo = null` after executed?